README.md

```diff
@@ -39,6 +39,7 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 
 - [Shouldly](https://github.com/shouldly/shouldly/labels/Jump-In) _(label: Jump-In)_ <br> Should testing for .net - the way Asserting *Should* be!
 - [MvvmCross](https://github.com/MvvmCross/MvvmCross/labels/first-timers-only) _(label: first-timers-only)_ <br> The .NET MVVM framework for cross-platform solutions, including Xamarin.iOS, Xamarin.Android, Windows and Mac.
+- [RawCMS](https://github.com/arduosoft/RawCMS/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) _(label: good first issue)_ <br> RawCMS is a headless CMS written in ASP.NET Core, built for developers that embrace API-first technology.
```

Comment:
I believe the link here should be https://github.com/arduosoft/RawCMS/labels/good%20first%20issue -  a direct link to the label

Reply:
Changed.